src/models/userRoleExtension.js

```diff
@@ -8,11 +8,11 @@
 module.exports = {
 	name: 'userRoleExtension',
 	schema: {
-		userRoleId: {
+		value: {
```

Comment:
@MallanagoudaB do not make changes to model. it is a confidential data. once we fix the model you can't change them, revert this change